crypto/txsigner.go

```diff
@@ -21,6 +22,9 @@ type TxSigner interface {
 
 	// SignTx signs a transaction
 	SignTx(tx *types.Transaction, priv *ecdsa.PrivateKey) (*types.Transaction, error)
+
+	// calculateV calculates the V value based on the type of signer used
+	calculateV(parity byte) []byte
```

Comment:
If all the implementations of the TxSigner interface are contained inside this package, then it's all good. @zivkovicmilos though I don't see why exactly you would intend that.

Reply:
This is definitely a method all signers should have, but since it's only used as part of local processing I didn't export it.

You're right to point out that if we have different implementations across packages, this wouldn't work as intended. 

Fixed, so it's exported :+1:

554e26a